tsinfer/inference.py

```diff
@@ -1856,6 +1893,57 @@ def __init__(self, sample_data, ancestors_ts, **kwargs):
         # node ID in the tree sequence.
         self.sample_id_map = {}
 
+    @staticmethod
+    def dask_find_path(
+        samples_slice,
+        tree_sequence_builder_wrapper=None,
+        data_path=None,
+        engine=None,
+        recombination=None,
+        mismatch=None,
+        precision=None,
+        extended_checks=None,
+        site_indexes=None,
+        sample_id_map=None,
+    ):
+        t = time_.time()
+        ancestor_matcher_class = (
+            _tsinfer.AncestorMatcher
+            if engine == constants.C_ENGINE
+            else algorithm.AncestorMatcher
+        )
+        matcher = ancestor_matcher_class(
+            tree_sequence_builder_wrapper.tsb,
+            recombination=recombination,
+            mismatch=mismatch,
+            precision=precision,
+            extended_checks=extended_checks,
+        )
+        sys.stderr.write(
```

Comment:
This is a bit grotty - a comment explaining why we need to do this (rather than logging) would be helpful. Also, why not ``print(..., file=sys.stderr)``?

Reply:
I had trouble getting dask to set the log level on the workers - just found https://github.com/dask/distributed/issues/6124 though which should fix that, so have changed the code to proper logging.